homeassistant/components/switch/fritzbox.py

```diff
@@ -0,0 +1,109 @@
+"""
+Support for AVM Fritz!Box smarthome switch devices.
+
+For more details about this component, please refer to the documentation at
+http://home-assistant.io/components/switch.fritzbox/
+"""
+import logging
+
+import requests
+
+from homeassistant.components.fritzbox import DOMAIN as FRITZBOX_DOMAIN
+from homeassistant.components.switch import SwitchDevice
+from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS
+
+DEPENDENCIES = ['fritzbox']
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_CURRENT_CONSUMPTION = 'current_consumption'
```

Comment:
There is already a standard attribute in the switch base class corresponding to this attribute here. We could make that change now, since switching platforms will be a breaking change anyway.